app/views/bus_stops/_form.haml

```diff
@@ -21,6 +18,18 @@
       %tr
         %td= f.label :completed, 'Completed?'
         %td= f.check_box :completed
+      %tr
+        %td= f.label :garage_responsible
+        %td= f.select :garage_responsible, %w[SATCo UMTS VATCo], include_blank: true
+      %tr
+        %td= f.label :state_road
+        %td= f.check_box :state_road
+      %tr
+        %td= f.label :date_stop_checked
+        %td= f.date_select :date_stop_checked, value: Date.today, disabled: true
```

Comment:
@mboneil10 my interpretation is that these ought to be autofilled by the controller when (and only when) someone is marking the stop as completed.

Is that your understanding?

Reply:
Yes, my fault I forgot to tell him to look at that issue.

Reply:
@jforgue, in #20 we concluded to do what @dfaulken said above. You can remove `:date_stop_checked` and `:stop_checked_by`. Sorry I didn't realize the confusion earlier.

Reply:
Will do, no worries.

Reply:
Do we just want the fields removed from the form or the attributes removed from the model? I assume the later as we mention `paper_trail` in the issue.

Reply:
As we talked about, both of these attributes can be covered with PaperTrail `versions` and `whodunnit`, so I'd remove the columns entirely at least for now.

Reply:
We'd discussed that that could have performance implications, so I agree to use PaperTrail for now, but we should keep the current approach in our back pocket.